src/index.js

```diff
@@ -1,11 +1,10 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById("root")
```

Comment:
Might be out of scope for now, but it would be cool to have linting in place to allow single quotes instead of double quotes.

Reply:
Good one, will put that on my todo list!

Reply:
+ prettier config